src/components/com_tjcertificate/site/views/certificate/view.html.php

```diff
@@ -69,6 +71,10 @@ public function display($tpl = null)
 			return false;
 		}
 
+		// Get HTML
+		$model = TJCERT::model('Certificate', array('ignore_request' => true));
+		$this->contentHtml = $model->getContentInfo($this->certificate->getClientId(), $this->certificate->getClient());
```

Comment:
use o/p from all plugins of content for the trigger used, and not just one

look at com_content code for reference